webscraping/webscrape.py

```diff
@@ -32,16 +35,51 @@ def requestHTML(reqName):
         file.close()
     return OUTPUT
```

Comment:
Should we return data1 here?

Reply:
it should be formatted_output yes, though I seem to have adjusted for it later in code. I can make changes to have it all work better together!